src/Lkl.cc

```diff
@@ -661,16 +670,23 @@ Double_t Lkl::CallMinimization(Double_t g,Bool_t isVerbose,Int_t strategy)
 
       // try changing precision
       if(iflag!=0 || TMath::IsNaN(GetParErr(gGParIndex)))
-	fMinuit->DefineParameter(gGParIndex,fParName[gGParIndex],g,fParDelta[gGParIndex]/10.*TMath::Power(1.8,counter), 0, 0);
```

Comment:
Same comment as above, related to the special treatment for all parameters called "tau"

Reply:
Agreed!